tests/driver_lis3mdl/README.md

```diff
@@ -0,0 +1,11 @@
+# About
+This is a manual test application for the LIS3MDL magnetometer driver.
+
+# Usage
+This test application will initialize the accelerometer with the following parameters:
+ - Sampling Rate: 10Hz
+ - Scale:         4G
+ - Temperature sensor: Enabled
```

Comment:
ups, never mind. Overlooked the reading of temp values at the bottom of the main...